### PR TITLE
Shils/link cleanup

### PR DESCRIPTION
**Pull request by shils**
Switches to https URLs in README.md and CONTRIBUTING.md wherever possible, and fixes a broken link to the dev forum.

---
This change is [Reviewable](https://reviewable.io/reviews/spockframework/spock/1133)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -3,31 +3,31 @@
...
+Good communication makes a big difference. We are always eager to listen, reflect, and discuss. Don't hesitate to get in touch via the [issue tracker](http://issues.spockframework.org), [user forum](http://forum.spockframework.org), or [dev forum](http://dev-forum.spockframework.org). Choose whatever medium feels most appropriate.
 
 ## Triage Issues [![Open Source Helpers](https://www.codetriage.com/spockframework/spock/badges/users.svg)](https://www.codetriage.com/spockframework/spock)
 
 You can triage issues which may include reproducing bug reports or asking for vital information, such as version numbers or reproduction instructions. If you would like to start triaging issues, one easy way to get started is to [subscribe to spock on CodeTriage](https://www.codetriage.com/spockframework/spock).
 
 ## Contributing Code/Docs
 
-To contribute code or documentation, please submit a pull request to the [GitHub repository](http://github.spockframework.org/spock).
+To contribute code or documentation, please submit a pull request to the [GitHub repository](https://github.spockframework.org/spock).
```
Comment (szpak) [inline CONTRIBUTING.md]: That link with HTTPS returns:
image

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -62,7 +62,7 @@ All tests are written in Spock. Tests for `spock-core` are located in the `spock
 
 ### Command Line Build
 
-Spock is built with [Gradle](http://www.gradle.org). The only prerequsite for executing the build is an installation of JDK 1.6 (or higher). After cloning the [GitHub repository](http://github.spockframework.org/spock), cd into the top directory and execute `./gradlew build` (Windows: `gradlew build`). The build should succeed without any errors. `gradlew tasks` lists the available tasks. Always use the Gradle Wrapper (`gradlew` command) rather than your own Gradle installation.
+Spock is built with [Gradle](https://www.gradle.org). The only prerequsite for executing the build is an installation of JDK 1.6 (or higher). After cloning the [GitHub repository](https://github.spockframework.org/spock), cd into the top directory and execute `./gradlew build` (Windows: `gradlew build`). The build should succeed without any errors. `gradlew tasks` lists the available tasks. Always use the Gradle Wrapper (`gradlew` command) rather than your own Gradle installation.
```
Comment (szpak) [inline CONTRIBUTING.md]: The same here with `https://github.spockframework.org/spock`

File: README.md [hunk 3]
```diff
@@ -52,9 +52,9 @@ Modules
 -------
 * spock-core -- Core framework. This is the only mandatory module.
 * spock-specs -- Specifications for spock-core, implemented using Spock. Not required for using Spock.
-* spock-spring -- Integration with the [Spring TestContext Framework](http://docs.spring.io/spring/docs/4.1.5.RELEASE/spring-framework-reference/html/testing.html#testcontext-framework).
-* spock-tapestry -- Integration with the [Tapestry 5](http://tapestry.apache.org/tapestry5/) IoC container.
-* spock-guice -- Integration with [Guice](http://code.google.com/p/google-guice/) 2/3.
+* spock-spring -- Integration with the [Spring TestContext Framework](https://docs.spring.io/spring/docs/4.1.5.RELEASE/spring-framework-reference/html/testing.html#testcontext-framework).
+* spock-tapestry -- Integration with the [Tapestry 5](https://tapestry.apache.org/tapestry5/) IoC container.
+* spock-guice -- Integration with [Guice](https://code.google.com/p/google-guice/) 2/3.
```
Comment (szpak) [inline README.md]: This could be updated to https://github.com/google/guice

File: README.md [hunk 4]
```diff
@@ -52,9 +52,9 @@ Modules
 -------
 * spock-core -- Core framework. This is the only mandatory module.
 * spock-specs -- Specifications for spock-core, implemented using Spock. Not required for using Spock.
-* spock-spring -- Integration with the [Spring TestContext Framework](http://docs.spring.io/spring/docs/4.1.5.RELEASE/spring-framework-reference/html/testing.html#testcontext-framework).
-* spock-tapestry -- Integration with the [Tapestry 5](http://tapestry.apache.org/tapestry5/) IoC container.
-* spock-guice -- Integration with [Guice](http://code.google.com/p/google-guice/) 2/3.
+* spock-spring -- Integration with the [Spring TestContext Framework](https://docs.spring.io/spring/docs/4.1.5.RELEASE/spring-framework-reference/html/testing.html#testcontext-framework).
+* spock-tapestry -- Integration with the [Tapestry 5](https://tapestry.apache.org/tapestry5/) IoC container.
```
Comment (szpak) [inline README.md]: https://tapestry.apache.org/ would be enough

File: CONTRIBUTING.md (hunk 1 above)
Comment (shils) [reply CONTRIBUTING.md]: Ah I mixed it up with https://github.com/spockframework, which that link redirects to. Would it be alright if I replace the link with https://github.com/spockframework?

File: README.md (hunk 4 above)
Comment (shils) [reply README.md]: https://github.com/spockframework/spock/pull/1133/commits/7d260955a8f2088945e5000818914e2bd1f6cb0d

File: README.md (hunk 3 above)
Comment (shils) [reply README.md]: https://github.com/spockframework/spock/pull/1133/commits/b84a3cbdb92f6d23cbba15955a2b5a74c0fbe695

File: CONTRIBUTING.md (hunk 1 above)
Comment (shils) [reply CONTRIBUTING.md]: Updated this and the link in the 'Command Line Build' section - https://github.com/spockframework/spock/pull/1133/commits/50dbf1460d10a740a73fc213e44dae4a4e65b8c9

**Comment by szpak** [comment]
Merged, thanks.

**Pull request merged**